overlay.d/07c9s/bootstrap/pre-pivot.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -euxo pipefail
+
+# This script is called by the OKD installer and runs in the
+# local context of the bootstrap node.
+# It rebases the booted operating system (e.g. Fedora CoreOS)
+# to CentOS Stream CoreOS 9
+
+# Load common functions
+. /usr/local/bin/bootstrap-service-record.sh
```

Comment:
?

Reply:
This is run by the installer in the context of the bootstrap node: see https://github.com/openshift/installer/blob/d10d64c9843e37f500bcb8d66caf339e940b918f/data/data/bootstrap/files/usr/local/bin/bootstrap-pivot.sh#L27